usr/share/rear/output/ISO/Linux-i386/700_create_efibootimg.sh

```diff
@@ -44,7 +44,41 @@ mount $v -o loop -t vfat $TMP_DIR/efiboot.img $TMP_DIR/efi_virt || Error "Failed
 # Copy files from staging directory into efiboot.img
 cp $v -r $TMP_DIR/mnt/. $TMP_DIR/efi_virt
 
-umount $v $TMP_DIR/efiboot.img
+# Umounting the EFI virtual image:
+local what_is_mounted="EFI virtual image $TMP_DIR/efiboot.img at $TMP_DIR/efi_virt"
+if ! umount $v $TMP_DIR/efiboot.img ; then
+    # Normal umounting something directly after some I/O command (like 'cp' above)
+    # may sometimes fail with "target is busy" (cf. 'busy' and 'lazy' in "man umount")
+    # so we retry after one second to increase likelihood that it then succeeds
+    # because normal umount is preferred over more sophisticated attempts
+    # like lazy or enforced umount which raise their own specific troubles:
+    Log "Failed to umount $what_is_mounted (will retry after one second)"
+    sleep 1
+    if ! umount $v $TMP_DIR/efiboot.img ; then
+        Log "Again failed to umount $what_is_mounted"
+        Log "$what_is_mounted is still in use by ('kernel mount' is always there)"
+        # The -M option avoids that fuser may show all processes using the '/' filesystem
+        # ( $TMP_DIR is $BUILD_DIR/tmp which is /var/tmp/rear.XXXXXXXXXXXXXXX/tmp/ )
+        # when $TMP_DIR/efiboot.img got umounted just before fuser starts, see "man fuser":
+        #   The mount -m option will match any file within the same device as the specified file,
+        #   use the -M option as well if you mean to specify only the mount point.
+        # So when $TMP_DIR/efiboot.img is umounted 'fuser -v -M -m $TMP_DIR/efi_virt' only shows
+        #   "Specified filename /var/tmp/rear.XXXXXXXXXXXXXXX/tmp/efi_virt is not a mountpoint"
+        # instead of all processes using '/' (or /var/ or /var/tmp/ if one is a mountpoint)
+        # which would be misleading information that may even look scaring and cause false alarm:
+        fuser -v -M -m $TMP_DIR/efi_virt 1>&2
```

Comment:
See https://github.com/rear/rear/issues/2908#issuecomment-1429605894 - older systems do not support `-M` - but since the code path where it is used is exceptional and the output is used only for information purposes, we don't care that much?

Reply:
`fuser` will probably fail if `-M` is not supported, which will hide the actual result that we are interested in.

Maybe `lsof` is more portable?

Reply:
I would like to keep our currently tested 'fuser ... -M ...'
and I agree with the @pcahyna reasoning in his above
https://github.com/rear/rear/pull/2909#discussion_r1105698584

See also
https://github.com/rear/rear/issues/2908#issuecomment-1476124509